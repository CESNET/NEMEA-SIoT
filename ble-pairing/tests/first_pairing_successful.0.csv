time TIMESTAMP,macaddr DEV_ADDR,macaddr HCI_DEV_ADDR,uint16 SIZE,uint8 DATA_DIRECTION,uint8 PACKET_TYPE,bytes PACKET
2019-10-20T20:59:45.910,00:00:00:00:00:00,5c:f3:70:87:46:2b,10,0,1,0b200700600030000001
2019-10-20T20:59:45.974,00:00:00:00:00:00,5c:f3:70:87:46:2b,6,1,4,0e04010b2000
2019-10-20T20:59:45.974,00:00:00:00:00:00,5c:f3:70:87:46:2b,5,0,1,0c20020101
2019-10-20T20:59:45.977,00:00:00:00:00:00,5c:f3:70:87:46:2b,6,1,4,0e04010c2000
2019-10-20T20:59:48.759,00:00:00:00:00:00,5c:f3:70:87:46:2b,45,1,4,3e2b020100013fd66be7e7d71f0201061bff5701005394b6956b4ec367ab502b9a76728faa01d7e7e76bd63fbc
2019-10-20T20:59:48.760,00:00:00:00:00:00,5c:f3:70:87:46:2b,5,0,1,0c20020000
2019-10-20T20:59:48.762,00:00:00:00:00:00,5c:f3:70:87:46:2b,6,1,4,0e04010c2000
2019-10-20T20:59:48.762,00:00:00:00:00:00,5c:f3:70:87:46:2b,28,0,1,0d20196000600000013fd66be7e7d7001800280000002a0000000000
2019-10-20T20:59:48.764,00:00:00:00:00:00,5c:f3:70:87:46:2b,6,1,4,0f0400010d20
2019-10-20T20:59:49.830,00:00:00:00:00:00,5c:f3:70:87:46:2b,21,1,4,3e130100400000013fd66be7e7d7270000002a0005
2019-10-20T20:59:49.830,00:00:00:00:00:00,5c:f3:70:87:46:2b,5,0,1,1620024000
2019-10-20T20:59:49.833,00:00:00:00:00:00,5c:f3:70:87:46:2b,6,1,4,0f0400011620
2019-10-20T20:59:49.861,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,20,1,2,402010000c00050012a50800180028000000f401
2019-10-20T20:59:49.910,00:00:00:00:00:00,5c:f3:70:87:46:2b,14,1,4,3e0c040040000100000000000000
2019-10-20T20:59:49.911,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,15,0,2,40000b00070006000104002d100d0f
2019-10-20T20:59:49.911,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,14,0,2,40000a000600050013a502000000
2019-10-20T20:59:49.911,00:00:00:00:00:00,5c:f3:70:87:46:2b,17,0,1,13200e4000180028000000f40100000000
2019-10-20T20:59:49.913,00:00:00:00:00:00,5c:f3:70:87:46:2b,6,1,4,0f0400011320
2019-10-20T20:59:50.008,00:00:00:00:00:00,5c:f3:70:87:46:2b,7,1,4,13050140000200
2019-10-20T20:59:50.030,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,11,0,2,4000070003000400020502
2019-10-20T20:59:50.056,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,15,1,2,40200b000700060002030001100505
2019-10-20T20:59:50.056,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,25,0,2,400015001100060003d3d999827a16178cf9460a7223c036a5
2019-10-20T20:59:50.105,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,11,1,2,4020070003000400038000
2019-10-20T20:59:50.105,00:00:00:00:00:00,5c:f3:70:87:46:2b,7,1,4,13050140000200
2019-10-20T20:59:50.105,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,15,0,2,40000b0007000400100100ffff0028
2019-10-20T20:59:50.154,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,25,1,2,402015001100060003733beaf9c5a91231db2a486c56cbd1b1
2019-10-20T20:59:50.154,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,25,0,2,40001500110006000434d45afef64ee94ee2d493a68afedf19
2019-10-20T20:59:50.203,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,28,1,2,4020180014000400110601000700001808000b0001180c0016000a18
2019-10-20T20:59:50.203,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,15,0,2,40000b0007000400101700ffff0028
2019-10-20T20:59:50.203,00:00:00:00:00:00,5c:f3:70:87:46:2b,7,1,4,13050140000200
2019-10-20T20:59:50.300,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,25,1,2,402015001100060004943c354120fb976af3ebdc2c9398e61a
2019-10-20T20:59:50.300,00:00:00:00:00:00,5c:f3:70:87:46:2b,31,0,1,19201c4000000000000000000000005a7107b804e9858660f5d59cee60b13b
2019-10-20T20:59:50.301,00:00:00:00:00:00,5c:f3:70:87:46:2b,12,1,4,3e0a0300400027000000f401
2019-10-20T20:59:50.302,00:00:00:00:00:00,5c:f3:70:87:46:2b,6,1,4,0f0400011920
2019-10-20T20:59:50.349,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,30,1,2,40201a0016000400111417001c00000710af090018211235000030150000
2019-10-20T20:59:50.349,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,15,0,2,40000b0007000400101d00ffff0028
2019-10-20T20:59:50.545,00:00:00:00:00:00,5c:f3:70:87:46:2b,7,1,4,13050140000100
2019-10-20T20:59:50.641,00:00:00:00:00:00,5c:f3:70:87:46:2b,6,1,4,080400400001
2019-10-20T20:59:50.641,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,25,1,2,4020150011000600069ae59d2cd84b0b02f6115938f43787ce
2019-10-20T20:59:50.642,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,19,1,2,40200f000b00060007eaffbe3961a03cdf0fb6
2019-10-20T20:59:50.643,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,25,1,2,40201500110006000ad2bd95c490a3831a2ee951d0ac8fffe6
2019-10-20T20:59:50.643,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,25,0,2,40001500110006000606d96e046e9c43f2e1ed6924509308f0
2019-10-20T20:59:50.643,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,19,0,2,40000f000b00060007d3517af7ae705303d374
2019-10-20T20:59:50.643,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,25,0,2,40001500110006000a78eaa262dbc32b4d2568a4c1931e5ea8
2019-10-20T20:59:50.690,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,31,1,2,40201b002000040011061d002300111824002600021827002c000d182d0051
2019-10-20T20:59:50.691,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,13,1,2,4010090000e0fe52006600e1fe
2019-10-20T20:59:50.691,00:00:00:00:00:00,5c:f3:70:87:46:2b,7,1,4,13050140000200
2019-10-20T20:59:50.692,00:00:00:00:00:00,5c:f3:70:87:46:2b,7,1,4,13050140000200
2019-10-20T20:59:50.797,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,15,0,2,40000b0007000400106700ffff0028
2019-10-20T20:59:50.885,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,13,1,2,40200900050004000110ffff0a
2019-10-20T20:59:50.938,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,13,0,2,4000090005000400120b000200
2019-10-20T20:59:50.983,00:00:00:00:00:00,5c:f3:70:87:46:2b,7,1,4,13050140000200
2019-10-20T20:59:51.031,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,9,1,2,402005000100040013
2019-10-20T20:59:51.031,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,11,0,2,40000700030004000a0300
2019-10-20T20:59:51.129,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,18,1,2,40200e000a0004000b4d492042616e642032
2019-10-20T20:59:51.129,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,11,0,2,40000700030004000a0500
2019-10-20T20:59:51.179,00:00:00:00:00:00,5c:f3:70:87:46:2b,7,1,4,13050140000200
2019-10-20T20:59:51.226,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,11,1,2,40200700030004000b0000
2019-10-20T20:59:51.226,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,11,0,2,40000700030004000a1600
2019-10-20T20:59:51.324,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,16,1,2,40200c00080004000b01570104000001
2019-10-20T20:59:51.419,00:00:00:00:00:00,5c:f3:70:87:46:2b,7,1,4,13050140000100
