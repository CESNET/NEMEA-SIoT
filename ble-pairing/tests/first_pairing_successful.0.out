2019-10-20T20:59:50.641,5c:f3:70:87:46:2b,d7:e7:e7:6b:d6:3f,0,0,1,0,"First pairing of device d7:e7:e7:6b:d6:3f on hci dev 5c:f3:70:87:46:2b"
